src/risk-risiko.py

```diff
@@ -27,3 +27,20 @@
 #   M 3 vs 3 => blue win
 #   O 2 vs 1 => red win
 #
+import random
+red_lost = 0
+blue_lost = 0
+red_dices = [random.randint(1,6) , random.randint(1,6) , random.randint(1,6)]
+red_dices.sort(reverse=True)
+blue_dices = [random.randint(1,6) , random.randint(1,6) , random.randint(1,6)]
+blue_dices.sort(reverse=True)
+print("Risultati dadi attacco:" , red_dices)
+print("Risultati dadi difesa:" , blue_dices)
+for i in range(3):
+    if blue_dices[i] >= red_dices[i]:
+        print(red_dices[i] , "vs" , blue_dices[i] , ": vince la difesa")
+        red_lost += 1
+    else:
+        print(red_dices[i] , "vs" , blue_dices[i] , ": vince l'attacco")
+        blue_lost += 1
+print("perdite:" , red_lost , "unità perse dall'attaccante," , blue_lost , "unità perse dalla difesa")
```

Comment:
potresti aggiungere un po' spazi e ritorni a capo nel codice per renderlo più leggibile

inoltre, potresti spezzare il codice in più funzioni